teps/0074-deprecate-pipelineresources.md

```diff
@@ -326,9 +314,7 @@ without requiring PipelineResources:
 
 ## Test Plan
 
-Since this is about removing an existing feature, not exactly sure what to put in the test plan. Once we start working
```

Comment:
I think the deleted bit is here still valid - we'd still need a way of supporting v1beta1 with resources even after we have done the v1 release before that?

Reply:
updated the note here